taxonomies/origins.txt

```diff
@@ -5695,11 +5825,388 @@ hu: Florida
 nl: Florida
 pl: Floryda
 
+< en:united-states
+en: Georgia, Georgia in the usa, Georgia in the united states, Georgia state, US-GA
```

Comment:
We already have en: Georgia in countries.txt, so we can't have just "Georgia" in origins.txt, or the entries will be combined. I suggest putting "Georgia USA" as the main name. Or "Georgia State".

Reply:
The State of Georgia would be best: https://en.wikipedia.org/wiki/Georgia_(U.S._state)